amplify/backend/types/amplify-dependent-resources-ref.d.ts

```diff
@@ -1,10 +1,11 @@
 export type AmplifyDependentResourcesAttributes = {
     "function": {
-        "expressServer": {
+        "newsArticleScrapingAndSentimentAnalysis": {
```

Comment:
Is this overwriting or adding ? im sure it's ok anyway just said i'd draw your attention to it to make sure.

Reply:
Im actually not sure look at your one too for your lambda its actually different to mine as well

Reply:
Fixed this if can have a look at latest commit

Reply:
![image](https://user-images.githubusercontent.com/61060096/195388808-b8aa0b37-0b4d-48ca-abaa-85575c051909.png)